OpenRA.Game/Network/OrderManager.cs

```diff
@@ -49,6 +49,7 @@ public sealed class OrderManager : IDisposable
 		readonly List<TextNotification> notificationsCache = new List<TextNotification>();
 
 		public IReadOnlyList<TextNotification> NotificationsCache => notificationsCache;
+		public long ChatDisabledUntil { get; internal set; }
```

Comment:
IMO this doesn't belong on the OrderManager.

Reply:
Neither, but IMO it does belong with the `NotificationsCache` (i.e. the chat history). Theres no rush to merge this PR, so we can wait until your PR that you mentioned splits that out.

Reply:
We'll need to come up with a solution to move this forward so we can get it in for the playtest.